main.js

```diff
@@ -51,9 +51,21 @@ const args = yargs
 const settingsPathJSON = path.join(__dirname, "settings.json");
 const settingsPathYAML = args.config;
 migrateSettingsToYAML(settingsPathJSON, settingsPathYAML);
-
-// Get the settings
-const rawSettingsObj = jsYaml.safeLoad(fs.readFileSync(settingsPathYAML));
+ 
+// Get the settings from a function
+function getSettings(){
+	try {
+		return jsYaml.safeLoad(fs.readFileSync(settingsPathYAML));
+	} catch (err){
+		if (err.code === "ENOENT") {
+			// We might be in a container, try to account for that and try again
```

Comment:
I would almost say there should be some form of feedback here getting written to stdout that indicates the normal settings.yaml is missing. I'd have put in a call to logger.warn() but logger is initialized with the settings this function is reading.